multi-agent/adapters/inbound/flask/endpoints/collaboration.py

```
@@ -0,0 +1,215 @@
+@collaboration_bp.route("/team.sessions", methods=["GET"])
+@from_query({
+    "team_id": fields.Str(data_key="teamId", required=True),
+})
+def get_team_sessions(team_id):
+    svc, err = service_for_team(team_id)
+    if err:
+        return err
+    try:
+        index = svc.get_team_sessions(team_id)
+        return jsonify(index.model_dump(mode="json")), 200
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)